dask_ml/model_selection/_incremental.py

```diff
@@ -614,7 +667,15 @@ def fit(self, X, y=None, **fit_params):
         **fit_params
             Additional partial fit keyword arguments for the estimator.
         """
-        return default_client().sync(self._fit, X, y, **fit_params)
+        if self.verbose:
+            h = logging.StreamHandler(sys.stdout)
+            context = LoggingContext(logger, level=logging.INFO, handler=h)
+            self._logging_context = context  # for testing ease; not necessary
```

Comment:
Let's avoid adding this just for testing. There are other ways to test that logging occurred: https://docs.pytest.org/en/latest/logging.html#caplog-fixture